ogc-custom-test/src/test/java/org/orbisgis/ogccustomtest/OGC_TestFilterEncoding.java

```diff
@@ -0,0 +1,179 @@
+        JAXBElement notEqualElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        comparisonOps = (FilterType) notEqualElement.getValue();
+
+        JAXBElement<BinaryComparisonOpType> notEqual = (JAXBElement<BinaryComparisonOpType>) comparisonOps.getComparisonOps();
+
+        Assert.assertEquals(notEqualElement.getName().getLocalPart(), "Filter");
+        Assert.assertEquals(notEqual.getName().getLocalPart(), "PropertyIsNotEqualTo");
+
+
+        //test PropertyIsLessThan
+        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_PropertyIsLessThan.xml");
+        JAXBElement isLessElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        comparisonOps = (FilterType) isLessElement.getValue();
+
+        JAXBElement<BinaryComparisonOpType> isless = (JAXBElement<BinaryComparisonOpType>) comparisonOps.getComparisonOps();
+
+        Assert.assertEquals(isLessElement.getName().getLocalPart(), "Filter");
+        Assert.assertEquals(isless.getName().getLocalPart(), "PropertyIsLessThan");
+
+
+        //test PropertyIsGreaterThan
+        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_PropertyIsGreaterThan.xml");
+        JAXBElement isGreaterElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        comparisonOps = (FilterType) isGreaterElement.getValue();
+
+        JAXBElement<BinaryComparisonOpType> isGreater = (JAXBElement<BinaryComparisonOpType>) comparisonOps.getComparisonOps();
+
+        Assert.assertEquals(isGreaterElement.getName().getLocalPart(), "Filter");
+        Assert.assertEquals(isGreater.getName().getLocalPart(), "PropertyIsGreaterThan");
+
+
+        //test PropertyIsGreaterThanOrEqualTo
+        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_PropertyIsGreaterThanOrEqualTo.xml");
+        JAXBElement isGreaterOrElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        comparisonOps = (FilterType) isGreaterOrElement.getValue();
+
+        JAXBElement<BinaryComparisonOpType> isGreaterOr = (JAXBElement<BinaryComparisonOpType>) comparisonOps.getComparisonOps();
+
+        Assert.assertEquals(isGreaterOrElement.getName().getLocalPart(), "Filter");
+        Assert.assertEquals(isGreaterOr.getName().getLocalPart(), "PropertyIsGreaterThanOrEqualTo");
+
+        //test PropertyIsLessThanOrEqualTo
+        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_PropertyIsLessThanOrEqualTo.xml");
+        JAXBElement isLessOrElement = (JAXBElement) unmarshaller.unmarshal(xml);
+        comparisonOps = (FilterType) isLessOrElement.getValue();
+
+        JAXBElement<BinaryComparisonOpType> isLessOr = (JAXBElement<BinaryComparisonOpType>) comparisonOps.getComparisonOps();
+
+        Assert.assertEquals(isGreaterOrElement.getName().getLocalPart(), "Filter");
+        Assert.assertEquals(isLessOr.getName().getLocalPart(), "PropertyIsLessThanOrEqualTo");
+
+
+
+    }
+
+    @Test
+    public void TestStandardFilter(){
+
```

Comment:
If this test is empty because it would be implemented soon, please add a comment to tell something like `To be implmented`